packages/dht/protos/DhtRpc.proto

```diff
@@ -164,6 +164,7 @@ message RouteMessageWrapper {
   Message message = 4;
   repeated PeerDescriptor reachableThrough = 5;
   repeated PeerDescriptor routingPath = 6;
+  repeated string parallelRootNodeIds = 7;
```

Comment:
nodeIds are sent as bytes in protocol level. Can this be e.g. array of bytes?

Reply:
Due to added complexity from binary comparison issues I'd rather just keep this as a list of strings